src/util/initEvents.ts

```diff
@@ -20,6 +21,8 @@ export const initEvents = () => {
     store.dispatch({ type: 'setCursor', thoughtsRanked, replaceContextViews: contextViews })
   })
 
+  window.addEventListener('mousemove', () => store.dispatch(toggleToolbarAndBreadCrumbs(true)))
```

Comment:
It would be more efficient to throttle outside the dispatch. Any reason why it was moved inside?

Reply:
The throttle has been moved to the [action-creator](https://github.com/cybersemics/em/blob/c954efc5607f85cac8e251d0ccc60d16b09cf3f8/src/action-creators/toggleToolbarAndBreadCrumbs.ts) to handle this [issue](https://github.com/cybersemics/em/pull/763#issuecomment-653035806), where the key-press and `mousemove` events are fired together. It's to ensure that only the last action gets called in an event where multiple actions are dispatched within a certain timespan (100ms in this case)

Reply:
Okay. Can you look for another way to do it? I'm really not comfortable dispatching an action on every mousemove trigger.

Reply:
This is a thunk being dispatched, and [this check](https://github.com/cybersemics/em/blob/c954efc5607f85cac8e251d0ccc60d16b09cf3f8/src/action-creators/toggleToolbarAndBreadCrumbs.ts#L9) within the action-creator ensures that we don't fire too many actions. Let me know if that addresses your concern. 
The other way of handling it could be wrapping [this](https://github.com/cybersemics/em/blob/4b2284367269839d48f752c438aa4e7e3c94014f/src/util/initEvents.ts#L24) function with a `debounce`.

Reply:
> This is a thunk being dispatched, and this check within the action-creator ensures that we don't fire too many actions. Let me know if that addresses your concern.

No, I don't care for this approach. Too many thunks being dispatched unnecessarily.

> The other way of handling it could be wrapping this function with a debounce and set fire it on the leading edge, which would ensure that only a single dispatch gets called right when the mouse starts to move, instead of getting called too frequently while the mouse is being moved.

Yes, I would appreciate this other approach.